core/src/com/unciv/logic/map/mapgenerator/MapLandmassGenerator.kt

```diff
@@ -312,6 +357,49 @@ class MapLandmassGenerator(val ruleset: Ruleset, val randomness: MapGenerationRa
         return 1.0 - (5.0 * shouldBeWater*shouldBeWater + randomScale) / 3.0
     }
 
+    private fun getOceanEdgesTransform(tile: Tile, tileMap: TileMap): Double {
+        val a = tileMap.maxLongitude
+        val b = tileMap.maxLatitude
+        val x = tile.longitude
+        val y = tile.latitude
+
+        var elevationOffset = 0.0
+
+        val xdistanceratio = abs(x)/a
+        val ydistanceratio = abs(y)/b
+        if (tileMap.mapParameters.shape === MapShape.hexagonal || tileMap.mapParameters.shape === MapShape.flatEarth) {
+            val startdropoff = 0.8
```

Comment:
What are these measurements? Long/lat? Percentages? Please add meaningful names.
You can't have elevationOffest and a uncategorized offset, either